### PR TITLE
DIT-1025 - proposed change for ops request sample

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/uk/gov/hmrc/bindingtariffclassification/model/Case.scala [hunk 1]
```diff
@@ -35,5 +35,7 @@ case class Case
   decision: Option[Decision] = None,
   attachments: Seq[Attachment] = Seq.empty,
   keywords: Set[String] = Set.empty,
-  sampleStatus: Option[SampleStatus] = None
+  sampleStatus: Option[SampleStatus] = None,
+  sampleRequested: Option[Boolean] = None,
+  sampleToBeReturned: Option[Boolean] = None
```
Comment (ed0906) [inline app/uk/gov/hmrc/bindingtariffclassification/model/Case.scala]: We have these already on the BTIApplication

Comment (ed0906) [reply app/uk/gov/hmrc/bindingtariffclassification/model/Case.scala]: Unless this is sample requested by HMRC staff?

**Comment by SteveSugden** [comment]
It is indeed - sample requested by HMRC.  We agreed (I think) not to "mess" with the applicants sending sample answer

**Comment by ed0906** [comment]
Do we need them to be `Option`s?

**Comment by SteveSugden** [comment]
See my own comments.   The first one could be a boolean, but the second we need three states - yes/no/don't know

**Comment by ed0906** [comment]
> See my own comments. The first one could be a boolean, but the second we need three states - yes/no/don't know

Is `false` not the same as `dont know`?

**Pull request closed without merging**